tests/utils/test_logging.py

```diff
@@ -24,3 +25,24 @@ def test_get_logger() -> None:
     assert isinstance(logger, Logger)
     assert log_file == str(expected_log_file)
     assert logger.name == __name__
+
+def test_reocurring_messages() -> None:
+    """Ensure logger filters out recocurring messages"""
+    logger = get_logger(__name__)
+    log_file = logger.handlers[0].baseFilename
+    expected_log_file = default_logsdir() / ("telliot-feed-examples.log")
+    expected_log_file.resolve().absolute()
+    num_lines_before = 0
+    with open(os.path.join(expected_log_file), "r") as f:
+        for line in f:
+            num_lines_before += 1
+
+    for _ in range(5):
+        logger.info("Address not yet staked. Depositing stake.")
```

Comment:
Maybe change the message to something unique for the tests? That, or do this in a tests.log file

Reply:
Ok